src/augmentations.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import random
+import cv2
+
+def to_numpy_img(img):
+    """From tensor to numpy, with channels last"""
+
+    img = img.data.numpy()
+    img = np.moveaxis(img, 0, -1)
+
+    return img
+
+def rotate(img, mask, rot=(-10, 10), p=0.5):
```

Comment:
Is there a reason not to use the transforms in torchvision?

https://pytorch.org/docs/stable/torchvision/transforms.html

Reply:
12 channel input

Reply:
Ah, okay, makes sense.